src/main/scala/mesosphere/marathon/api/v2/json/Formats.scala

```diff
@@ -27,6 +27,8 @@ import scala.concurrent.duration._
 // https://mesosphere.atlassian.net/browse/MARATHON-1292
 object Formats extends Formats {
 
+  type LightweightDeploymentPlanFlag = Boolean
```

Comment:
_According to https://phabricator.mesosphere.com/D1056 discussion:_
* This type should be removed and the `Boolean` now flag should be inlined